### PR TITLE
hierarchies-react: Prepare for release

### PR DESCRIPTION
**Pull request by grigasp**
Closes https://github.com/iTwin/presentation/issues/422

### CODE REVIEW

File: packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx [hunk 1]
```diff
@@ -8,49 +8,60 @@ import cx from "classnames";
...
   onNodeClick,
   onNodeKeyDown,
-  setHierarchyLevelLimit,
   isSelected,
   isDisabled,
   actionButtonsClassName,
+  getHierarchyLevelDetails,
   ...nodeProps
 }: TreeNodeRendererProps) {
+  if ("type" in node && node.type === "ChildrenPlaceholder") {
```
Comment (saskliutas) [inline packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx:52]: Why was this moved before `isPresentationHierarchyNode` check?

File: packages/hierarchies-react/README.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+- `selectionMode` is a string that defines the selection mode. It can be one of the following values:
+  - `none` - no selection is allowed,
+  - `single` - only one node can be selected at a time,
+  - `extended` - multiple nodes can be selected using shift and ctrl keys,
+  - `multiple` - multiple nodes can be selected without using shift or ctrl keys.
+
+The returned object contains 2 functions, that should be called by the node renderer: `onNodeClick` and `onNodeKeyDown`.
+
+Out [tree renderer implementation](#treerenderer) calls this hook and passes the callbacks to the [node renderer](#treenoderenderer), so there's no need to use it unless implementing a custom tree renderer.
```
Comment (saskliutas) [inline packages/hierarchies-react/README.md:77]:
```suggestion
Our [tree renderer implementation](#treerenderer) calls this hook and passes the callbacks to the [node renderer](#treenoderenderer), so there's no need to use it unless implementing a custom tree renderer.
```

**Review by saskliutas — APPROVED** [review]

File: packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx (hunk 1 above)
Comment (grigasp) [reply packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx:52]: because the "ChildrenPlaceholder" type of not is not a `PresentationHierarchyNode` anymore

**Review by jasdom — APPROVED** [review]
